undocked/tools/export_program_info_sample/export_program_info_sample.vcxproj

```diff
@@ -243,4 +243,4 @@ $(OutputPath)export_program_info_sample.exe</Command>
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-</Project>
+</Project>
```

Comment:
Why this change?

Reply:
Editor stripped new lines from end of file for some reason.